### DISCUSSION TITLE
Always choose best available quality

### DISCUSSION THREAD

**Discussion by volme** [FreeTubeApp/FreeTube#4207] [General]
Hi, Is there any way too change options about video quality to always choosing the best available video quality by default? In player settings i can choose only 1080p as highest possible.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
If your computer can handle it, you can enable the Allow AV1 Formats setting. That will use the AV1 formats when they are available (YouTube doesn't provide them for most videos), which have 1440p and 4k resolutions.

The current video player we are using, video.js, doesn't support WebM DASH, so we cannot support the VP9 formats which are available on most videos and have 1440p and 4k resolutions (of course really old videos that were uploaded before YouTube supported VP9, but I'm sure there are some special cases that have been re-encoded in the meantime).

Thats why u can only select 1080p in that menu.

**Reply by volme** [reply]
I have WebM DASH enabled and i am able to choose 4k, 1440p in player. So its not possible to set player to choose best available quality by default?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
No because of the restrictions of the player we're using.

**Reply by absidue** [reply]
@volme FreeTube doesn't support WebM DASH, because the player that we use, video.js, doesn't support it, so it is not possible for you to enable it.

I'm guessing you mean the Allow AV1 setting?
